.github/workflows/pr-checks.yaml

```diff
@@ -24,5 +24,5 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-rust
 
-    - name: Rust checks
-      run: make check-all-dockerized
+    - name: DRink! tests
+      run: make all-drink-dockerized
```

Comment:
Does it mean that CI won't run unit tests now, or am I mistaken?

Reply:
Changed the `all-drink` to also run `check-all`. Good eye.